swap_meet/vendor.py

```diff
@@ -1,2 +1,76 @@
 class Vendor:
-    pass
\ No newline at end of file
+
+    def __init__(self, inventory=None):
+        self.inventory = inventory if inventory is not None else []
+
+    def add(self, item):
+        self.inventory.append(item) # This method adds the item to the inventory
+        return item
```

Comment:
An optional enhancement could be, to check if the `item` that is trying to be added to a vendor's `inventory` is already in there.